hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -378,6 +396,7 @@ public Builder() {
       this.metadata = new HashMap<>();
       this.quotaInBytes = OzoneConsts.QUOTA_RESET;
       this.quotaInNamespace = OzoneConsts.QUOTA_RESET;
+      this.bucketType = BucketType.DEFAULT;
```

Comment:
Can you elaborate on the BucketType.DEFAULT -> OBJECT_STORE? Should we chose LEGACY for backward compatibility?

Reply:
New users will only be able to create FILE_SYSTEM_OPTIMIZED and OBJECT_STORE buckets. Only existing buckets can be of type LEGACY. In BucketType.java we have mentioned that if the bucket type is null then the bucket type will be initialized to LEGACY. Also for unit tests, the buckets which don't have bucket type will be initialized to LEGACY, then we will check if the layout is of type prefix then we will change the bucket type to FILE_SYSTEM_OPTIMIZED. In the follow up jiras we will remove the layout 'PREFIX' concept. I hope this answers your question!